src/Core/Core.csproj

```diff
@@ -10,12 +10,9 @@
     <AssemblyName>SonarLint.VisualStudio.Core</AssemblyName>
 
     <RequiresNewtonsoft>true</RequiresNewtonsoft>
+    <RequiresSystemIOAbstractions>true</RequiresSystemIOAbstractions>
```

Comment:
All of the affected project files have these changes i.e. add the new property, and remove the explicit package reference.